srcpkgs/sssd/template

```diff
@@ -29,6 +29,7 @@ homepage="https://sssd.io"
 distfiles="https://github.com/SSSD/sssd/releases/download/${pkgname}-${version//./_}/${pkgname}-${version}.tar.gz"
 checksum=ef8b047e6d0452a585862dffd16db725ac828e0d3fb594a8dea6d2f24a61ad17
 python_version=3
+nocross="configure attempts to run compiled output"
```

Comment:
How did this succeed previously?

Reply:
It didn't; the dependency `ldb` was previously `nocross`.